airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -232,7 +233,15 @@ def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Ite
             # to limit the number of API Calls and reduce the time of data fetch,
             # we can pull the ready data for child_substream, if nested data is present,
             # and corresponds to the data of child_substream we need.
-            if self.nested_substream:
+            if self.nested_substream and self.nested_substream_list_field_id:
+                if record.get(self.nested_substream):
+                    sorted_substream_slices += [
```

Comment:
let's use `.append()` instead of the math operation `+=`, this is pretty obvious why you've used it, but let's keep the code consistent)